sdk/search/azure-search-documents/azure/search/documents/_service/_search_service_client.py

```diff
@@ -64,6 +65,24 @@ def __repr__(self):
         # type: () -> str
         return "<SearchServiceClient [endpoint={}]>".format(repr(self._endpoint))[:1024]
 
+    def __enter__(self):
+        # type: () -> SearchServiceClient
+        self._client.__enter__()  # pylint:disable=no-member
+        return self
+
+    def __exit__(self, *args):
+        # type: (*Any) -> None
+        self._client.__exit__(*args)  # pylint:disable=no-member
```

Comment:
Technically, this should be `return self._client.__exit__(*args)`, no? I don't *think* our generated clients would ever return `True` from their `__exit__` method, so it may not matter right now.

Reply:
the generated client stipulates a None return:
```
    def __exit__(self, *exc_details):
        # type: (Any) -> None
        self._client.__exit__(*exc_details)
```
But I am happy to add a return to "future proof" things

Reply:
fixed in https://github.com/Azure/azure-sdk-for-python/pull/10831/commits/7f8c7ff8f2438cba1992624b1f6ac50d486d04bf